### PR TITLE
Configured the follower motor

### PR DESCRIPTION
**Pull request by imathno**

### CODE REVIEW

File: Competition2017/src/competition/subsystems/shooter_wheel/ShooterWheelSubsystem.java [hunk 1]
```diff
@@ -62,12 +62,15 @@ public ShooterWheelSubsystem(int motor, RobotSide side, WPIFactory factory, XPro
...
         enableShooterLogging = propManager.createEphemeralProperty("Is " + side + "Shooter logging info?", false);
         
-        this.masterMotor = factory.getCANTalonSpeedController(motor);
-        setUpMotorTeam(masterMotor);
-        masterMotor.createTelemetryProperties(side + "Shooter master", propManager);
+        this.masterMotor = factory.getCANTalonSpeedController(masterChannel);
+        this.followerMotor = factory.getCANTalonSpeedController(followerChannel);
+        
+        setUpMotorTeam(masterMotor, followerMotor);
+        masterMotor.createTelemetryProperties(side + "Shooter wheel master", propManager);
```
Comment (WasabiFan) [inline Competition2017/src/competition/subsystems/shooter_wheel/ShooterWheelSubsystem.java]: Add a space in the string between the number and "shooter" (also, decapitalize "Shooter")

File: Competition2017/src/competition/subsystems/shooter_wheel/ShooterWheelSubsystem.java [hunk 2]
```diff
@@ -82,11 +85,15 @@ private void setUpMotorTeam(XCANTalon master) {
...
-        master.setF(.099);
         updateMotorPidValues(master);
         master.setControlMode(TalonControlMode.Speed);
-
         master.set(0);
+        
+        follower.setProfile(0);
+        follower.setControlMode(TalonControlMode.Follower);
+        follower.configNominalOutputVoltage(0,  -0);
+        follower.configPeakOutputVoltage(12, -6);
```
Comment (WasabiFan) [inline Competition2017/src/competition/subsystems/shooter_wheel/ShooterWheelSubsystem.java:96]: I believe the follower needs to spin backwards relative to the master. That should be configured, and additionally these values may need to be swapped. Or they could probably just be removed, given that the master constraints should apply here too.

File: Competition2017/src/competition/subsystems/shooter_wheel/ShooterWheelSubsystem.java (hunk 1 above)
Comment (WasabiFan) [inline Competition2017/src/competition/subsystems/shooter_wheel/ShooterWheelSubsystem.java]: Does something update the telemetry?

Comment (imathno) [reply Competition2017/src/competition/subsystems/shooter_wheel/ShooterWheelSubsystem.java]: "but it can buy me a boat" done

Comment (imathno) [reply Competition2017/src/competition/subsystems/shooter_wheel/ShooterWheelSubsystem.java]: done

File: Competition2017/src/competition/subsystems/shooter_wheel/ShooterWheelSubsystem.java (hunk 2 above)
Comment (imathno) [reply Competition2017/src/competition/subsystems/shooter_wheel/ShooterWheelSubsystem.java:96]: done

File: Competition2017/src/competition/subsystems/shooter_wheel/ShooterWheelSubsystem.java [hunk 3]
```diff
@@ -82,11 +86,15 @@ private void setUpMotorTeam(XCANTalon master) {
...
-        master.setF(.099);
         updateMotorPidValues(master);
         master.setControlMode(TalonControlMode.Speed);
-
         master.set(0);
+        
+        follower.setProfile(0);
+        follower.setControlMode(TalonControlMode.Follower);
+        follower.configNominalOutputVoltage(0,  -0);
+        follower.configPeakOutputVoltage(-12, 6);
```
Comment (JohnGilb) [inline Competition2017/src/competition/subsystems/shooter_wheel/ShooterWheelSubsystem.java]: I'm pretty sure this also needs to be `12, -6` based on some earlier readthroughs of the CANTalon code,

**Review by JohnGilb — CHANGES_REQUESTED** [review]

File: Competition2017/src/competition/subsystems/shooter_wheel/ShooterWheelSubsystem.java (hunk 2 above)
Comment (aschokking) [reply Competition2017/src/competition/subsystems/shooter_wheel/ShooterWheelSubsystem.java:96]: Is the currently physically modeled config the same as the 2016 summer bot shooter?

File: Competition2017/src/competition/subsystems/shooter_wheel/ShooterWheelSubsystem.java (hunk 3 above)
Comment (JohnGilb) [reply Competition2017/src/competition/subsystems/shooter_wheel/ShooterWheelSubsystem.java]: I think the proper way to invert this is to call `follower.setInverted(true)`, though we should verify that actually works on-robot =]

**Review by JohnGilb — APPROVED** [review]

**Pull request merged**